### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by granitn**
its something

### CODE REVIEW

**Comment by mzschunke** [comment]
test

**Comment by doctype-Melvin** [comment]
Wahnsinn!

File: github-and-markdown.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## github command
+
+- git status = wich branch
+- git add = put file on stage
+- git add . = whole directory to stage
+- git commit = commit staged files
+  - an -m " message"
+- git log = what has been commited
```
Comment (doctype-Melvin) [inline github-and-markdown.md:8]: git loggggg

**Review by doctype-Melvin — COMMENTED** [review]
Another comment. Good Job!

**Pull request merged**